src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/rules/VerifyServiceProviderAppRule.java

```diff
@@ -6,12 +6,27 @@
 import uk.gov.ida.saml.core.IdaSamlBootstrap;
 import uk.gov.ida.verifyserviceprovider.VerifyServiceProviderApplication;
 import uk.gov.ida.verifyserviceprovider.configuration.VerifyServiceProviderConfiguration;
-
 import static uk.gov.ida.saml.core.test.TestCertificateStrings.TEST_RP_PRIVATE_ENCRYPTION_KEY;
 import static uk.gov.ida.saml.core.test.TestCertificateStrings.TEST_RP_PRIVATE_SIGNING_KEY;
 
 public class VerifyServiceProviderAppRule extends DropwizardAppRule<VerifyServiceProviderConfiguration> {
 
+
+    public VerifyServiceProviderAppRule(String secondaryEncryptionKey, String serviceEntityIdOverride) {
+        super(
+            VerifyServiceProviderApplication.class,
+            "verify-service-provider.yml",
+            ConfigOverride.config("serviceEntityIds", serviceEntityIdOverride),
+            ConfigOverride.config("hashingEntityId", "some-hashing-entity-id"),
+            ConfigOverride.config("server.connector.port", String.valueOf(0)),
+            ConfigOverride.config("logging.loggers.uk\\.gov", "DEBUG"),
+            ConfigOverride.config("samlSigningKey", TEST_RP_PRIVATE_SIGNING_KEY),
+            ConfigOverride.config("verifyHubConfiguration.environment", "COMPLIANCE_TOOL"),
+            ConfigOverride.config("samlPrimaryEncryptionKey", TEST_RP_PRIVATE_ENCRYPTION_KEY),
+            ConfigOverride.config("samlSecondaryEncryptionKey", secondaryEncryptionKey),
+            ConfigOverride.config("msaMetadata.uri", "http://some-msa-uri")
```

Comment:
`ConfigOverride.config("msaMetadata.uri", "http://some-msa-uri")` is this supposed to go too. I tried to remove this line and I started to get a null Exception

Reply:
So after discussing with Andy, this is currently out of scope for this card.